packages/calypso-build/webpack/sass.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+const MiniCssExtractPluginWithRTL = require( 'mini-css-extract-plugin-with-rtl' );
+
+module.exports.loader = ( { preserveCssCustomProperties, includePaths, prelude } ) => ( {
+	test: /\.(sc|sa|c)ss$/,
+		use: [
+		MiniCssExtractPluginWithRTL.loader,
+		{
+			loader: 'css-loader',
```

Comment:
we may need to wrap these in `require.resolve` to make them reliable. webpack uses the normal node_modules package resolution strategy to find imports, but since this config is being pulled into a larger config that lives in a different execution context, the resolution rules may fail to find the module. Right now this only really works because npm hoists dependencies to the root. This probably won't work with yarn.

Imagine a layout like this:
```
webpack.config.js
node_modules/
  @automattic/calypso-build/
    webpack/sass.js
    node_modules/css-loader
```
if webpack.config.js tried to find css-loader, it will not, it's hidden under @automattic/calypso-build. 

To make this reliable, we can use require.resolve to always resolve to the closest `css-loader`.

Reply:
1ab261027f